src/App.tsx

```diff
@@ -1,14 +1,21 @@
 import { useState } from "react";
+import { Route, RouterProvider, createBrowserRouter, createRoutesFromElements } from "react-router-dom";
 import "./App.css";
 import SplashScreen from "./components/SplashScreen";
-
+import Layout from "./pages/Layout";
+import PaniniCreatorLayout from "./pages/PaniniCreatorLayout";
 function App() {
   const [orderData, setOrderData] = useState({});
-  return (
-    <div>
-      <SplashScreen isPaniniOrdered={Object.keys(orderData).length > 0 ? true : false}></SplashScreen>
-    </div>
+  const router = createBrowserRouter(
+    createRoutesFromElements(
+      <Route path="/" element={<Layout isPaniniOrdered={Object.keys(orderData).length > 0 ? true : false} />}>
+        <Route index element={<SplashScreen shouldTransition={false} />} />
+        <Route path="/panini_creator" element={<PaniniCreatorLayout shouldTransition={true} />}></Route>
+        <Route path="/panini_creator:options" element={<PaniniCreatorLayout shouldTransition={false} />}></Route>
```

Comment:
**question:** Why do you need this route?

Reply:
# Explanation: 
About the options route, I've figured out that the state of form (order) would be preserved in the url (so the user could share it with friends, and come back to his configuration with just url, so it's kinda future functionality. 
Also for now, the animation turns on automatically without any params, so i've figured out such workaround,
# Thoughts: 
 Maybe better approach would be to navigate user (from splashscreen) to path with :transitioned_param for transition animation? And then create subRoute for order (with inputs customised by user, through url)?